lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -301,17 +301,11 @@ html.video-fullscreen {
           &.fa-arrow-circle-right {
             top: 30%;
           }
-        }
-        &:last-child {
-          > .fa {
+          &.exam-without-proctoring {
```

Comment:
In fact, I think I'll switch the logic and have a ``primary-action`` class instead...